src/main/resources/application.yaml

```diff
@@ -364,8 +364,11 @@ notifications:
   notifyApplicantForHwfNoRemission: "3008a18a-07ec-445b-a226-62ffa19132ce"
   notifyApplicantForHwfNoRemissionWelsh: "f9d2f0db-2ae4-4691-8076-bfbb82548846"
   notifyApplicantForHwFMoreInformationNeeded: "02956398-fffb-4461-93a0-3713788363a8"
+  notifyApplicantForHwFMoreInformationNeededWelsh: "65b3524b-b58c-4cdb-b8b5-5fc2cec505a5"
   notifyApplicantForHwfUpdateRefNumber: "7f87e629-6473-4732-8d80-4174d06696d5"
+  notifyApplicantForHwfUpdateRefNumberBilingual: "7f87e629-6473-4732-8d80-4174d06696d5"
```

Comment:
This property has been added but I don't think is currently being used.  Also it would be good to be consistent with the naming - we should either use "...Bilingual" or "...Welsh" but not both.

Reply:
It is added to civil-commons for another ticket, so I needed to ad for tests to run.
for naming, I agree we should align on one of them